docs/administration-guide/20-custom-maps.md

```diff
@@ -0,0 +1,25 @@
+## Custom Maps
+
+By default, Metabase uses OpenStreetMaps for map visualizations, but there are a few customization options.
+
+### Finding Map Settings
+
+To find the map settings:
+
+1. Click the gear icon in the upper right.
+2. Select `Maps` from the navigation on the left.
+
+![Map Settings](images/MapSettings.png)
+
+### Map Tile Server
+
+If you'd like to use a map tile server other than OpenStreetMaps, simply add the map tile server URL to the first field on the page.
+
+### Custom Maps
```

Comment:
I think we could distinguish region maps from world/pin maps a bit more explicitly. This is also the second "Custom Maps" heading on this page, so I'd suggest changing this to "Custom Region Maps".